### PR TITLE
Command line attribute provider

### PR DESCRIPTION
**Pull request by konraddysput**
# Why

This diff adds an additional way to determinate the application package.json path. This time, this diff adds support to node.js  files executed via command lines like node ./file.js

### CODE REVIEW

File: packages/node/src/attributes/ApplicationInformationAttributeProvider.ts [hunk 1]
```diff
@@ -50,6 +50,10 @@ export class ApplicationInformationAttributeProvider implements BacktraceAttribu
 
     private generateDefaultApplicationSearchPaths() {
         const possibleSourcePaths = [process.cwd()];
+        const potentialCommandLineStartupFile = process.argv[1];
+        if (potentialCommandLineStartupFile && fs.existsSync(potentialCommandLineStartupFile)) {
+            possibleSourcePaths.unshift(potentialCommandLineStartupFile);
```
Comment (perf2711) [inline packages/node/src/attributes/ApplicationInformationAttributeProvider.ts:55]: Can we `path.resolve` this path before returning it?

Comment (adamcronin42) [reply packages/node/src/attributes/ApplicationInformationAttributeProvider.ts:55]: For my learning, why do we need an absolute path?

**Review by adamcronin42 — APPROVED** [review]

File: packages/node/src/attributes/ApplicationInformationAttributeProvider.ts (hunk 1 above)
Comment (konraddysput) [reply packages/node/src/attributes/ApplicationInformationAttributeProvider.ts:55]: the relative path generated by the backtrace-node library might point to a different place if sdk-core starts using it.

File: packages/node/src/attributes/ApplicationInformationAttributeProvider.ts [hunk 2]
```diff
@@ -51,8 +51,11 @@ export class ApplicationInformationAttributeProvider implements BacktraceAttribu
     private generateDefaultApplicationSearchPaths() {
         const possibleSourcePaths = [process.cwd()];
         const potentialCommandLineStartupFile = process.argv[1];
-        if (potentialCommandLineStartupFile && fs.existsSync(potentialCommandLineStartupFile)) {
-            possibleSourcePaths.unshift(potentialCommandLineStartupFile);
+        if (potentialCommandLineStartupFile) {
+            const potentialCommandLineStartupFilePath = path.resolve(potentialCommandLineStartupFile);
+            if (fs.existsSync(potentialCommandLineStartupFilePath)) {
+                possibleSourcePaths.unshift(path.resolve(potentialCommandLineStartupFilePath));
+            }
```
Comment (perf2711) [inline packages/node/src/attributes/ApplicationInformationAttributeProvider.ts:58]: Why are you resolving this twice?

**Review by perf2711 — APPROVED** [review]

**Pull request merged**